templates/missed_landing_comment.html

```diff
@@ -15,7 +15,7 @@
                 <a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ bugid }}">{{ bugid }}</a>
             </td>
             <td>
-                {% for cset in csets %}<a href="{{ cset[1] | e }}">{{ cset[0] }}</a>{% endfor %}
+                {% for cset in csets %}<a href="{{ cset[1] | e }}">{{ cset[0] }}</a><br/>{% endfor %}
```

Comment:
```suggestion
                <ul style="margin: 0">
                    {% for cset in csets %}
                        <li>
                            <a href="{{ cset[1] | e }}">{{ cset[0] }}</a>
                        </li>
                    {% endfor %}
                </ul>
```

Reply:
I'm not sure bullets would be useful here?

Reply:
I agree with you that bullet would not be helpful. I suggested using a list to avoid having a blank line at the end. We could add `list-style-type:none;` to the `style` attribute to remove them.

Reply:
Ah, yeah, that's fair, I didn't think about the trailing line break; thanks.